🚨 Decrease number of warnings and suggestions (#2547)
platformsh/platformsh-docs: docs/src/security/updates.md

{{% description %}}

Platform.sh periodically updates its container images for the latest security updates from upstream providers.
(PHP versions, Ruby versions, MariaDB versions, etc.).

> These do not always happen immediately but when a security vulnerability is identified and released it tends to be fairly soon after.

Reviewer: :hammer_and_wrench:
```suggestion
These don't always happen immediately but when a security vulnerability is identified and released it tends to be fairly soon after.
```

Author: Out of scope
